### ISSUE TITLE
getModuleDirectorySlow java.lang.Throwable: Assertion failed

### ISSUE THREAD

**Issue by matteoopenf** [wt-io-it/odoo-pycharm-plugin#53]
Plugin Version: 0.5.16
IDEA Version: PyCharm 2022.1.2

<details><summary>java.lang.Throwable: Assertion failed</summary>

```
java.lang.Throwable: Assertion failed
	at com.intellij.openapi.diagnostic.Logger.assertTrue(Logger.java:218)
	...
	at com.intellij.psi.search.FilenameIndex.getFilesByName(FilenameIndex.java:79)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.lambda$getModuleDirectorySlow$3(OdooModuleServiceImpl.java:136)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:920)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.getModuleDirectorySlow(OdooModuleServiceImpl.java:134)
	at at.wtioit.intellij.plugins.odoo.modules.impl.OdooModuleServiceImpl.getModuleDirectory(OdooModuleServiceImpl.java:121)
	at at.wtioit.intellij.plugins.odoo.modules.index.OdooDeserializedModuleImpl.getDirectory(OdooDeserializedModuleImpl.java:64)
	at at.wtioit.intellij.plugins.odoo.WithinProject.call(WithinProject.java:15)
	at at.wtioit.intellij.plugins.odoo.pycharm.PyCharmOdooAddonsImportResolver.resolveImportReference(PyCharmOdooAddonsImportResolver.java:38)
	at at.wtioit.intellij.plugins.odoo.pycharm.PyCharmOdooAddonsImportResolver.resolveImportReference(PyCharmOdooAddonsImportResolver.java:42)
	at com.jetbrains.python.psi.resolve.PyResolveImportUtil$foreignResults$1.invoke(PyResolveImportUtil.kt:199)
	...
	at java.base/java.lang.Thread.run(Thread.java:829)
```
</details>

**Comment by ap-wtioit** [comment]
@matteoopenf this was fixed in 0.5.18, can you upgrade the plugin?

same as #33

**Comment by matteoopenf** [comment]
> @matteoopenf this was fixed in 0.5.18, can you upgrade the plugin?
> 
> same as #33

thank you I will do the upgrade
